rasa/nlu/featurizers/dense_featurizer/convert_featurizer.py

```diff
@@ -89,8 +88,11 @@ def create(
         model_storage: ModelStorage,
         resource: Resource,
         execution_context: ExecutionContext,
+        cache_dir: str = None,
     ) -> ConveRTFeaturizer:
         """Creates a new component (see parent class for full docstring)."""
+        if cache_dir:
```

Comment:
this should happen in the `__init__` method and be accessed through the config. This way it will always be applied no matter in which way the Featurizer instance is created